firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -151,11 +184,43 @@ bool moveAlongEdges(int startCol, int startRow, int endCol, int endRow)
   currCol = currPositionX / (stepsPerUnitSpace * 8);
   currRow = currPositionY / (stepsPerUnitSpace * 8);
 
+  // Print debug info about moveAlongEdges
```

Comment:
Can you make this comment and the comment in moveDirect have the same form? Here you use the function name, but there you say straight movement.